closed/autoconf/custom-hook.m4

```diff
@@ -193,13 +193,21 @@ AC_DEFUN_ONCE([OPENJ9_PLATFORM_SETUP],
   AC_ARG_WITH(noncompressedrefs, [AS_HELP_STRING([--with-noncompressedrefs],
       [build non-compressedrefs vm (large heap)])])
 
+  AC_ARG_WITH(valhalla_nestmates, [AS_HELP_STRING([--with-valhalla_nestmates],[build with valhalla nestmates spec])])
+
+  if test "x$with_valhalla_nestmates" != x; then
+    EXTENDED_SPEC="_valhalla_nestmates"
+  else
+    EXTENDED_SPEC=""
+  fi
+
   # When compiling natively host_cpu and build_cpu are the same. But when
   # cross compiling we need to work with the host_cpu (which is where the final
   # JVM will run).
   OPENJ9_PLATFORM_EXTRACT_VARS_FROM_CPU($host_cpu)
 
   if test "x$with_noncompressedrefs" = x; then
-    OPENJ9_BUILDSPEC="${OPENJDK_BUILD_OS}_${OPENJ9_CPU}_cmprssptrs"
+    OPENJ9_BUILDSPEC="${OPENJDK_BUILD_OS}_${OPENJ9_CPU}_cmprssptrs${EXTENDED_SPEC}"
```

Comment:
If valhalla is only going to support ppcle then I don't believe this line needs to be set, otherwise --with_valhalla_nestmates on x86 will yield a failure at some stage.  Does line 238, which sets the spec name for ppcle specifically, not work on its own?